### PR TITLE
Sagaz_dApp_PW_for_VeWorld

### PR DESCRIPTION
**Pull request by WildBillDev**
Thanks for all the hard work on your side!!

### CODE REVIEW

File: apps/com.thesagaz/manifest.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "name": "Sagaz",
+    "href": "https://thesagaz.com",
+    "desc": "Caelum AB Inferno",
+    "category": "collectibles",
+    "tags": [
+        "Sagaz",
+        "VeChain",
+        "Blockchain",
+        "VIP-181",
+        "VIP-20",
```
Comment (libotony) [inline apps/com.thesagaz/manifest.json:11]: `VIP-20` seem not a valid VIP

**Comment by WildBillDev** [comment]
oh k will drop that sorry that was for token my bad I will change that and repush

**Comment by WildBillDev** [comment]
got it fixed let me know if this works

**Comment by WildBillDev** [comment]
Any idea on timeline for this?

File: apps/com.thesagaz/manifest.json (hunk 1 above)
Comment (WildBillDev) [reply apps/com.thesagaz/manifest.json]: Made changes

**Comment by libotony** [comment]
@WildBillDev logo should be 512x512 in pixel size

**Comment by WildBillDev** [comment]
K i will make a resize

**Comment by WildBillDev** [comment]
Man i am sure having fun with this lol I think this should be good to go fingers crossed thank you libotony really appreciate you,
Much Love

File: apps/com.thesagaz/manifest.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        "PFP",
+        "Sassafras",
+        "SASS",
+        "Questing",
+        "Battles",
+        "Training"
+    ],
+    "contracts": [
+        "0xf46d132cfc813da70c0806921d05ab3057d8badd",
+        "0x88270ca93379de8245d653070d958ae28b91B01c",
```
Comment (libotony) [inline apps/com.thesagaz/manifest.json:22]: The contract address should be in lowercase.

Comment (WildBillDev) [reply apps/com.thesagaz/manifest.json:22]: Oh man I thought i got them in lower case one sec man I really sure am having a go with this lol I am so sorry for maken you come back to this over and over I feel bad

**Review by libotony — APPROVED** [review]

**Pull request merged**